Source/WebCore/html/HTMLScriptElement.cpp

```diff
@@ -87,9 +88,44 @@ void HTMLScriptElement::didFinishInsertingNode()
 }
 
 // https://html.spec.whatwg.org/multipage/scripting.html#dom-script-text
-void HTMLScriptElement::setText(String&& value)
+ExceptionOr<void> HTMLScriptElement::setText(std::variant<RefPtr<TrustedScript>, String>&& value)
 {
-    setTextContent(WTFMove(value));
+    ExceptionOr<String> stringValueHolder = trustedScriptCompliantStringFromVariant(*scriptExecutionContext(), WTFMove(value), "HTMLScriptElement text"_s);
```

Comment:
I suggest writing `auto` here instead of `ExceptionOr<String>` or refactoring so there is no local variable.